lib/modules/percentage.rb

```diff
@@ -0,0 +1,15 @@
+module Percentage 
+  def percent_of(value) 
+    (self.to_f/100) * value
+  end
+
+end
+
+class String
+  include Percentage 
```

Comment:
I'm fine with the monkey patching of the numeric class but strings should *definitely* not be working out percentages